### PR TITLE
Fix windows output, reopened to run appveyor

### LINKED ISSUE #40: node-speaker does not load in Windows 8
When requiring node-speaker, nothing happens. The program is stuck at the `require('speaker')`.
I'm using Windows 8.1 with Visual Studio 2013 and node 0.10.26

### PR DESCRIPTION
**Pull request by dfcreative**
This PR merges [RTK/node-speaker](https://github.com/RTK/node-speaker) and #49. Fixes #40.
@TooTallNate please have a look.

### CODE REVIEW

**Comment by dfcreative** [comment]
@TooTallNate yay, CI passed

**Comment by dfcreative** [comment]
@TooTallNate poke

**Comment by dfcreative** [comment]
@TooTallNate ping

**Comment by dfcreative** [comment]
@TooTallNate hello?

**Comment by dfcreative** [comment]
@TooTallNate please, merge

**Comment by dfcreative** [comment]
@TooTallNate just interested, is it ignore or notifications do not work.

**Comment by dfcreative** [comment]
@TooTallNate :sob:

**Comment by dfcreative** [comment]
[audio-speaker](https://github.com/audio-lab/audio-speaker) start getting likes, I want it to be dependent on npm, not github. @TooTallNate please

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "pulse",
     "mpg123"
   ],
-  "version": "0.2.6",
+  "license": "MIT",
+  "version": "0.2.7",
```
Comment (TooTallNate) [inline package.json]: Please don't bump the version number in the PR. That will come afterwards.

File: index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
 var os = require('os');
 var debug = require('debug')('speaker');
 var binding = require('bindings')('binding');
-var inherits = require('util').inherits;
+var inherits = require('inherits');
```
Comment (TooTallNate) [inline index.js]: Why this change?

**Comment by dfcreative** [comment]
@TooTallNate done
